diagnostic.md

```diff
@@ -40,25 +61,29 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
   ```md
-    # < Your Response Here >
+    The serializer lists the attributes associated with a table and it defines what information to obtain from a related table
```

Comment:
The serializer filters and reshapes the data before it's rendered as a response.